indico/modules/files/models/files.py

```diff
@@ -18,6 +17,8 @@
 from indico.util.fs import secure_filename
 from indico.util.string import format_repr, return_ascii, strict_unicode
 
+import posixpath
```

Comment:
CI is failing here. Not sure yet why this is happening. But please ignore it, I'll fix it before merging.

Reply:
> Not sure yet why this is happening. 

`posixpath` is stdlib` so that import should remain at its previous position